pkg/scheduler/objects/queue.go

```diff
@@ -41,29 +40,32 @@ import (
 	"github.com/apache/yunikorn-scheduler-interface/lib/go/common"
 )
 
+var (
+	maxPreemptionsPerQueue = 10 // maximum number of asks to attempt to preempt for in a single queue
```

Comment:
I made it a var because it's possible we may configure it later. I didn't want to do so right away, as I think this will work fine as a default until proven otherwise.